spec/integration/template_evaluation_spec.rb

```diff
@@ -55,6 +55,15 @@
     expect(new_id).to match /[a-f0-9\-]/
 
     expect(new_id).to eq(original_id)
+
+    output = bosh_runner.run('events')
+    parser = Support::TableHelpers::Parser.new(scrub_event_time(scrub_random_cids(scrub_random_ids(output))))
+    expect(parser.data).to include(
+      {'ID' => /[0-9]{1,3} <- [0-9]{1,3}/, 'Time' => 'xxx xxx xx xx:xx:xx UTC xxxx', 'User' => 'test', 'Action' => 'update', 'Object type' => 'deployment', 'Task' => /[0-9]{1,3}/, 'Object ID' => 'simple', 'Dep' => 'simple', 'Inst' => '-', 'Context' => '-'},
+      {'ID' => /[0-9]{1,3} <- [0-9]{1,3}/, 'Time' => 'xxx xxx xx xx:xx:xx UTC xxxx', 'User' => 'test', 'Action' => 'recreate', 'Object type' => 'instance', 'Task' => /[0-9]{1,3}/, 'Object ID' => 'id_job/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'Dep' => 'simple', 'Inst' => 'id_job/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'Context' => '-'},
+      {'ID' => /[0-9]{1,3}/, 'Time' => 'xxx xxx xx xx:xx:xx UTC xxxx', 'User' => 'test', 'Action' => 'recreate', 'Object type' => 'instance', 'Task' => /[0-9]{1,3}/, 'Object ID' => 'id_job/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'Dep' => 'simple', 'Inst' => 'id_job/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'Context' => '-'},
+      {'ID' => /[0-9]{1,3}/, 'Time' => 'xxx xxx xx xx:xx:xx UTC xxxx', 'User' => 'test', 'Action' => 'update', 'Object type' => 'deployment', 'Task' => /[0-9]{1,3}/, 'Object ID' => 'simple', 'Dep' => 'simple', 'Inst' => '-', 'Context' => '-'},
+    )
```

Comment:
When running this test, the events output includes the following seemingly-duplicate lines - two each for the `create` and `delete` event. It seems separate from the changes here, but curious if it's expected.

```
| 13 <- 12 | Wed Apr 27 00:03:12 UTC 2016 | test | create   | vm           | 13540                                       | 5    | simple | id_job/c50eeb84-fa56-40be-8755-33faa968b881 | -       |
| 12       | Wed Apr 27 00:03:12 UTC 2016 | test | create   | vm           | -                                           | 5    | simple | id_job/c50eeb84-fa56-40be-8755-33faa968b881 | -       |
| 11 <- 10 | Wed Apr 27 00:03:12 UTC 2016 | test | delete   | vm           | 13349                                       | 5    | simple | id_job/c50eeb84-fa56-40be-8755-33faa968b881 | -       |
| 10       | Wed Apr 27 00:03:12 UTC 2016 | test | delete   | vm           | 13349                                       | 5    | simple | id_job/c50eeb84-fa56-40be-8755-33faa968b881 | -       |
```